docs/en_US/Compression/Overview.rst

```diff
@@ -14,7 +14,9 @@ NNI provides a model compression toolkit to help user compress and speed up thei
 * Provide friendly and easy-to-use compression utilities for users to dive into the compression process and results.
 * Concise interface for users to customize their own compression algorithms.
 
-*Note that the interface and APIs are unified for both PyTorch and TensorFlow, currently only PyTorch version has been supported, TensorFlow version will be supported in future.*
+.. note::
+  NNI compresses the model through a simulation process. To obtain a truly compact model, users should conduct `model speedup <./ModelSpeedup.rst>`__. The interface and APIs are unified for both PyTorch and TensorFlow, currently only PyTorch version has been supported, TensorFlow version will be supported in future.
```

Comment:
Since NNI compression algorithms are not meant to compress model while NNI speedup tool can truly compress model and reduce latency . I think 'NNI compression algorithms simulate the the process of pruning and quantization without truly compressing model and reducing latency.' might be more appropriate at the beginning of this paragraph.

Reply:
Thanks, added.